x/module_accounts.md

```diff
@@ -0,0 +1,18 @@
+
+# Module Accounts
+
+All modules have their own module account. Think of this as a wallet that can only be controlled by that module. Below is a table of their respective wallet addresses and permissions.
+
+
+| Name                    | Address                                             | Permissions        |
```

Comment:
can you also add the  IBC `transfer` module account?

Reply:
Is this different from the `transfer` account `evmos1yl6hdjhmkf37639730gffanpzndzdpmhv788dt` ?

Reply:
ah no you are right